tests/iree-pytorch/default.nix

```diff
@@ -0,0 +1,111 @@
+{
+  linkerScript,
+  iree,
+  iree-turbine,
+  iree-runtime,
+  python3,
+  makeBuilder,
+  findAndBuild,
+  getTestRequiredFeatures,
+  t1main,
+  rtlDesignMetadata,
+}:
+
+let
+  builder = makeBuilder { casePrefix = "iree-pytorch"; };
+  device_embedded_sync-c = ./device_embedded_sync.c;
+  test-c = ./test.c;
+  pythonEnv = python3.withPackages (ps: [
+    ps.torch
+    iree-turbine
+  ]);
+  build =
+    { caseName, sourcePath }:
+    builder {
+      inherit caseName;
+
+      src = sourcePath;
+
+      passthru.featuresRequired = getTestRequiredFeatures sourcePath;
+
+      nativeBuildInputs = [
+        iree
+        pythonEnv
+      ];
+      buildInputs = [ iree-runtime ];
+
+      ireeCFlags = toString [
+        "-march=rv32imagc_zvl${toString rtlDesignMetadata.vlen}b_zve32f"
+        "-mabi=ilp32f"
```

Comment:
In the future, we may provide march in rtlDesignMetadata, avoid making it in every testcase. Any ideas? @Avimitin  

This comment is not blocking for this PR.